_guides/understandingmetrics.md

```diff
@@ -1,69 +1,76 @@
+
+Each model type may utilize different metrics and different metrics can tell you slightly different things about the same model. The Nexosis API returns several metrics for most models so that you can evaluate your model more fully. Let's look at each metric grouped by the model type.
+
+### <a name="regressionmetrics" class="jumptarget"></a> Forecast & Regression
+
+#### <a name="mae" class="jumptarget"></a>Mean Absolute Error
+Describes in absolute units the difference between the predicted and actual target values, on average for a regression or time series model. This metric does not penalize large prediction errors as heavily as Root Mean Squared Error.  Smaller is always better, but what constitutes a good results depends on the context of the specific problem. If you are predicting house prices a MAE of $500 is a very good result. But if you are predicting book prices that would constitute a bad result.
+
+
+#### <a name="mape" class="jumptarget"></a>Mean Absolute Percent Error
+Describes in percentage how far off the forecasts are from actuals, on average, for a time series model. This has the advantage that it is easily interpretable as a percentage error. However, this metric can be skewed if the target has values close to or equal to zero. In that case any error represents a large percentage error. Smaller is always better and as a rough rule of thumb, anything under 20% can probably be considered a good time series model.
+
+#### <a name="mase" class="jumptarget"></a>Mean Absolute Scaled Error
+This scale-free error metric can be used to compare forecast methods on a single time series and also to compare forecast accuracy between time series. This metric is well suited to intermittent-demand series because it never gives infinite or undefined values except in the irrelevant case where all historical data are equal.  Smaller values are better. If the MASE is less than 1 then the forecast error is better than the average one-step naive forecast, computed in-sample.
+
+
+#### <a name="rsquared" class="jumptarget"></a>R-Squared
+Measures how close the predicted target values are to the actual target values from a regression model, with a value typically between 0 and 1. (A very bad model can report negative values.) Values closer to 1 indicate that the model better explains the variability in the target values.
+
+#### <a name="rmse" class="jumptarget"></a>Root Mean Squared Error
+Describes in absolute units how far off the forecasts are from actuals, on average. This metric penalizes large prediction errors more heavily than Mean Absolute Error.  Smaller values are always better, but what constitutes a good result for RMSE depends on the context of the problem just like for MAE.

```

Comment:
I think saying absolute units is confusing. Squared units is more accurate or maybe some other wording